tensorflow_tts/processor/base_processor.py

```diff
@@ -61,11 +61,14 @@ def __post_init__(self):
         self.create_symbols()
         if self.saved_mapper_path is not None:
             self._save_mapper(saved_path=self.saved_mapper_path)
-        
+
         # processor name. usefull to use it for AutoProcessor
         self._processor_name = type(self).__name__
 
         if self.setup_eos_token():
+            self.add_symbol(
+                self.setup_eos_token()
+            )  # if this eos token not yet present in symbols list.
             self.eos_id = self.symbol_to_id[self.setup_eos_token()]
```

Comment:
This isn't needed __getattr__ should return eos_id properly

Reply:
That is the case eos symbol is 'eos' :))) how about if user use 'EOS' as eos token

Reply:
True, maybe just add extra tokens dict back and use it as last time (in case we need to add another extra tokens to processor?)

Reply:
Add a method add_extra_token -> which utils add_symbol and map extra token to class attr.
For example:

```python3
def add_extra_token(token_name: Union[str, list], symbol: Union[str, list]):
    if ...:
        self.extra_token[token_name] = symbol
        self.add_symbol(symbol)
  
```

Reply:
@machineko since we haven't write a doc for this framework yet so i think let user define their own 'eos' token is clearer than add extra_tokens :D. I add abstractmethod so user need to overide it :)).